goal_src/engine/ui/progress-h.gc

```diff
@@ -44,6 +44,8 @@
   (no-disc 32)
   (bad-disc 33)
   (quit 34)
+  ;; custom
+  (language-options 35)
```

Comment:
I think adding this is going to break the type consistency check.  I'd really rather not add in some feature to the compiler to ignore certain type error, so how about this idea?

We can keep `progress-h.gc` and other files in here unmodified. Then, we create a new folder like `goal_src/pc/` and put in our modified `progress-h.gc` there.  The type consistency check will look at the unmodified version. Then, you can edit the `game.gp` file to use `goal_src/pc/progress-h.gc` instead.   The type consistency check doesn't use the build system, so I think it will still pass.

Reply:
yeah, I like this idea a lot.  It also makes it obvious what we've heavily modified and what we havn't.  I'll do `pc/engine/ui/progress...` instead to preserve the file-tree.  Then its as simple as adding/removing `pc/` from the file paths to regression test